modules/perception/obstacle/camera/visualizer/frame_content.cc

```diff
@@ -34,25 +34,16 @@ FrameContent::~FrameContent() {}
 void FrameContent::set_image_content(double timestamp, cv::Mat image) {
   ImageContent image_content;
   image_content.timestamp_ = timestamp;
-
   image_content.image_mat_src_ = image;
   image_caches_[DoubleToMapKey(timestamp)] = image_content;
 }
 
-void FrameContent::set_motion_content(double timestamp,
-                                      MotionBufferPtr motion_buffer) {
-  MotionContent motion_content;
-  motion_content.motion_frame_content_ = *motion_buffer;
-  motion_caches_[DoubleToMapKey(timestamp)] = motion_content;
-}
-
 void FrameContent::set_camera_content(double timestamp,
                                       Eigen::Matrix4d pose_c2w,
                                       const std::vector<ObjectPtr>& objects,
                                       const CameraFrameSupplement& supplement) {
-  if (camera_caches_.find(DoubleToMapKey(timestamp)) != camera_caches_.end()) {
-    return;
-  }
+  auto key = DoubleToMapKey(timestamp);
+  if (camera_caches_.count(key)) return;
```

Comment:
Do you want to add debug log to this abnormal return for easier debug later? Similarly to later ones

Reply:
No problem. Will do later